### PR TITLE
Adding internal referral headers

### PR DESCRIPTION
**Pull request by andrewrlee**
…inks

### Context

https://trello.com/c/rJHDuj88/

### Intent

So we can distinguish between users guessing content urls vs broken navigation

Also updating/fixing audit

Adding request url to uncaught exception message

Extracting routes into own file

Removing unnecessary dependency

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -71,26 +71,25 @@
   "devDependencies": {
     "@sinonjs/fake-timers": "^7.1.2",
     "babel-eslint": "^10.1.0",
-    "chance": "^1.1.7",
```
Comment (andrewrlee) [inline package.json:74]: this was only used by a test util that was no longer being used

File: server/app.js [hunk 2]
```diff
@@ -102,6 +87,7 @@ const createApp = ({
   app.use(
     helmet({
       contentSecurityPolicy: false,
+      referrerPolicy: { policy: ['no-referrer', 'same-origin'] },
```
Comment (andrewrlee) [inline server/app.js:90]: default to no-referrer if same origin is not specfied.

same-origin should ensure allowlisted sites do not receive referral information

File: server/app.js [hunk 3]
```diff
@@ -330,7 +261,7 @@ const createApp = ({
 
   // eslint-disable-next-line no-unused-vars
   function renderErrors(error, req, res, next) {
-    logger.error(`Unhandled error - ${error.message}`);
+    logger.error(`Unhandled error - ${req.originalUrl} - ${error.message}`);
```
Comment (andrewrlee) [inline server/app.js:264]: adding url into messages to help track down source of errors

File: test/test-helpers.js [hunk 4]
```diff
@@ -21,20 +20,6 @@ const logger = {
...
-    title: chance.profession({ rank: true }),
-    contentType: chance.pickone(['radio', 'video', 'pdf', 'article']),
-    summary: chance.paragraph({ sentences: 1 }),
-    image: {
-      alt: chance.word(),
-      url: chance.avatar({ fileExtension: 'jpg' }),
-    },
-    contentUrl: `/content/${chance.integer()}`,
-  };
-}
```
Comment (andrewrlee) [inline test/test-helpers.js:36]: no longer used

**Review by leonk — APPROVED** [review]
Lgtm

**Review by davidthomas22 — APPROVED** [review]
LGTM

**Pull request merged**